nsq-cluster.gemspec

```diff
@@ -39,17 +23,14 @@ Gem::Specification.new do |s|
     s.specification_version = 4
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<sys-proctable>.freeze, [">= 0"])
-      s.add_development_dependency(%q<jeweler>.freeze, ["~> 2.2"])
+      s.add_runtime_dependency(%q<sys-proctable>.freeze, ["~> 1.2"])
```

Comment:
Should these `sys-proctable` entries be `~> 1.3`?

Reply:
The tests also consistently pass for me at 1.2, but I could change it to 1.3 to be consistent with what's in the gemfile.